Documentation/setup.md

```diff
@@ -0,0 +1,82 @@
+# EKS Factory Reference Guides
```

Comment:
@parkand1 looks like the majority of this overlaps with the existing readme. Do you want to add any updates/differences to the existing readme (so we don't have duplicate stuff for the time being)?

Reply:
+1, let's move pertinent parts to readme.

Reply:
Sounds good, will do